apps/hrrawexp/metadata.json

```diff
@@ -2,13 +2,14 @@
   "id": "hrrawexp",
   "name": "HRM Data Exporter",
   "shortName": "HRM Data Exporter",
-  "version": "0.03",
+  "version": "0.04",
   "description": "export raw hrm signal data to a csv file",
   "icon": "app-icon.png",
   "tags": "",
   "supports": ["BANGLEJS"],
```

Comment:
Looks like you've got a duplicate line here - maybe just update this line rather than adding `"supports": ["BANGLEJS","BANGLEJS2"]`?